add November newsletter (#609)
USRSE/usrse.github.io: _posts/newsletters/2021-11-30-newsletter.md

For more details on the survey visit [https://software.ac.uk/news/complete-international-rse-survey-and-help-us-understand-rse-community](https://software.ac.uk/news/complete-international-rse-survey-and-help-us-understand-rse-community).

You can complete the 2021 RSE survey in [English](https://softwaresaved.limequery.com/386272?lang=en), [French](https://softwaresaved.limequery.com/386272?lang=fr), [German](https://softwaresaved.limequery.com/386272?lang=de-informal), or [Spanish](https://softwaresaved.limequery.com/386272?lang=es).

<a name="workshop"></a>
# **US-RSE Community Workshop**

We are excited to announce the first US-RSE Association [community building workshop](https://us-rse.org/first-community-workshop/) April 26-27, 2022 in Princeton, NJ, after two years of delays due to COVID-19! This will be a workshop focused on planning the best path forward to grow the US-RSE Association. You can find out more and express interest in attending by filling out the **[US-RSE Workshop Interest Form](https://forms.gle/E5BKkysrfQZ2EtXbA)**.

<a name="community-calls"></a>
# **US-RSE Community Calls**

> The video for the November Community Call on "Current Technologies" is available at [https://youtu.be/nhbFizVEN7Y](https://youtu.be/nhbFizVEN7Y)

Reviewer: ```suggestion
The video for the November Community Call on "Current Technologies" is available on [YouTube](https://youtu.be/nhbFizVEN7Y)
```

> The video for the November Community Call on "Current Technologies" is available at [https://youtu.be/nhbFizVEN7Y](https://youtu.be/nhbFizVEN7Y)

Reviewer: ```suggestion
The video for the November Community Call on "Current Technologies" is available on [YouTube](https://youtu.be/nhbFizVEN7Y).
```

> All Community Call videos are listed in this playlist [https://www.youtube.com/playlist?list=PLMgG8jy8w-8neP6gPOgifH6kh4t_RUTTH](https://www.youtube.com/playlist?list=PLMgG8jy8w-8neP6gPOgifH6kh4t_RUTTH)

Reviewer: ```suggestion
All Community Call videos are listed in this [playlist](https://www.youtube.com/playlist?list=PLMgG8jy8w-8neP6gPOgifH6kh4t_RUTTH)
```

> All Community Call videos are listed in this playlist [https://www.youtube.com/playlist?list=PLMgG8jy8w-8neP6gPOgifH6kh4t_RUTTH](https://www.youtube.com/playlist?list=PLMgG8jy8w-8neP6gPOgifH6kh4t_RUTTH)

Reviewer: ```suggestion
All Community Call videos are listed in this [playlist](https://www.youtube.com/playlist?list=PLMgG8jy8w-8neP6gPOgifH6kh4t_RUTTH).
```

14. [Senior Software Manager for Data-intensive Astronomical Research Program](http://uwhires.admin.washington.edu/ENG/Candidates/default.cfm?szCategory=jobprofile&szOrderID=198215): Seattle (or Pittsburgh) *Posted: Nov 03, 2021*
15. [Senior Scientist Research Computing (Biomedical and Clinical Informatics)](https://jobs.rutgers.edu/postings/143690): Rutgers University, New Brunswick, NJ *Posted: Oct 27, 2021*
16. [Full Stack Software Engineer](https://jobs.ornl.gov/job/Oak-Ridge-Full-Stack-Software-Engineer-TN-37830/793415900/): Oak Ridge National Laboratory, Oak Ridge, TN *Posted: Oct 27, 2021*
17. [Full Stack Software Engineer](https://jobs.ornl.gov/job/Oak-Ridge-Full-Stack-Software-Engineer-TN-37830/793411000/): Oak Ridge National Laboratory, Oak Ridge, TN *Posted: Oct 27, 2021*
18. [Researcher](https://jobs.hr.wisc.edu/en-us/job/511080/researcher): University of Wisconsin-Madison, Madison, WI *Posted: Oct 22, 2021*
19. [Software Engineer](https://sjobs.brassring.com/TGnewUI/Search/home/HomeWithPreLoad?partnerid=25240&siteid=5341&PageType=JobDetails&jobid=1923828): Harvard T.H. Chan School of Public Health, Boston, MA *Posted: Sep 20, 2021*

> 20. [Research Software Engineer](https://uw-echospace.github.io/position/2021-09-06-rse/): University of Washington, Seattle, WA *Posted: Aug 10, 2021*

Reviewer: delete this one - it's not recent

<a name="news"></a>
# **Community News**
SC21-TV interviewed [Charles Ferenbaugh about the research software engineering landscape](https://www.youtube.com/watch?v=MjO5tEarTLU), and [Daniel S. Katz about research software sustainability](https://www.youtube.com/watch?v=1YODIJyX0t0).

> The videos from the [International RSE Day](https://researchsoftware.org/council/intl-rse-day.html) are available under [https://www.youtube.com/playlist?list=PLMgG8jy8w-8ng3nXVt1hpx3dhcejOd-kT](https://www.youtube.com/playlist?list=PLMgG8jy8w-8ng3nXVt1hpx3dhcejOd-kT).

Reviewer: ```suggestion
The videos from the [International RSE Day](https://researchsoftware.org/council/intl-rse-day.html) are available on [this YouTube playlist](https://www.youtube.com/playlist?list=PLMgG8jy8w-8ng3nXVt1hpx3dhcejOd-kT).
```

<a name="dei-mc"></a>
# **DEI Media Club Discussion**

> In an effort to promote continuing education and discussion, the [Diversity, Equity, and Inclusion Working Group (DEI-WG)](https://us-rse.org/about/working-groups/#diversity-equity-and-inclusion-dei) at the US-RSE Association presents the next instance of its DEI Media Club on December 2, 2021, 11:00 AM - 12:00 PM: a discussion of an episode of [Code Switch](https://www.npr.org/podcasts/510312/codeswitch): [The Folk Devil Made Me Do It](https://www.npr.org/2021/08/20/1029775224/the-folk-devil-made-me-do-it). [Sign up](https://emory.zoom.us/meeting/register/tJAqceCpqTopGtRj9YvJnWD9u3v7SZSdNCBb) to let us know that you are interested. To get most out of it: Come prepared to discuss!

Reviewer: ```suggestion
In an effort to promote continuing education and discussion, the US-RSE [Diversity, Equity, and Inclusion Working Group (DEI-WG)](https://us-rse.org/about/working-groups/#diversity-equity-and-inclusion-dei) presents the next instance of its DEI Media Club on December 2, 2021, 11:00 AM - 12:00 PM: a discussion of an episode of [Code Switch](https://www.npr.org/podcasts/510312/codeswitch): [The Folk Devil Made Me Do It](https://www.npr.org/2021/08/20/1029775224/the-folk-devil-made-me-do-it).

[Sign up](https://emory.zoom.us/meeting/register/tJAqceCpqTopGtRj9YvJnWD9u3v7SZSdNCBb) to let us know that you are interested.

To get most out of it: Come prepared to discuss!
```

Please share the event with your colleagues: [https://emory.zoom.us/meeting/register/tJAqceCpqTopGtRj9YvJnWD9u3v7SZSdNCBb](https://emory.zoom.us/meeting/register/tJAqceCpqTopGtRj9YvJnWD9u3v7SZSdNCBb)

<a name="events"></a>
# **Interesting Events**

Some other events coming up may be of interest to RSEs:

* The [AGU Fall meeting](https://www.agu.org/Fall-Meeting) takes place as a hybrid event December 13-17, 2021 (in person in New Orleans) and has sessions on software and open science such as [Advancing Notebooks as Scholarly Objects](https://agu.confex.com/agu/fm21/meetingapp.cgi/Session/124253).

> * [HICSS-55](https://hicss.hawaii.edu/) (55th Hawaii International Conference on System Sciences) goes fully virtual January 3-7, 2022. [Registration](https://hicss-submissions.org/) is required but free! The [Software Technology](https://hicss.hawaii.edu/tracks-55/software-technology/) track includes minitracks such as [Software Sustainability: Strategies for Long-Lasting and Usable Software](https://hicss.hawaii.edu/tracks-55/software-technology/#software-sustainability-strategies-for-long-lasting-and-usable-research-software-minitrack) and [Smart (City) and Data Streaming Application Development: Challenges and Experiences](https://hicss.hawaii.edu/tracks-55/software-technology/#smart-city-and-data-streaming-application-development-challenges-and-experiences-minitrack)**.**

Reviewer: ```suggestion
* [HICSS-55](https://hicss.hawaii.edu/) (55th Hawaii International Conference on System Sciences) is fully virtual ,January 3-7, 2022. [Registration](https://hicss-submissions.org/) is required but free! The [Software Technology](https://hicss.hawaii.edu/tracks-55/software-technology/) track includes minitracks such as [Software Sustainability: Strategies for Long-Lasting and Usable Software](https://hicss.hawaii.edu/tracks-55/software-technology/#software-sustainability-strategies-for-long-lasting-and-usable-research-software-minitrack) and [Smart (City) and Data Streaming Application Development: Challenges and Experiences](https://hicss.hawaii.edu/tracks-55/software-technology/#smart-city-and-data-streaming-application-development-challenges-and-experiences-minitrack)**.**
```

Reviewer: sorry, bad " ," that should be ", "

<a name="reads"></a>
# **Interesting Reads**

Items you may have missed on the blog and Slack:

* [The Missing Millions - Democratizing Computation and Data to Bridge Digital Divides and Increase Access to Science for Underrepresented Communities](https://www.rti.org/publication/missing-millions/fulltext.pdf)

* [What Works: Evidence-Based Ideas to Increase Diversity, Equity, and Inclusion in the Workplace](https://www.umass.edu/employmentequity/what-works-evidence-based-ideas-increase-diversity-equity-and-inclusion-workplace)

<a name="outreach"></a>
# **US-RSE Outreach**

Do you have a meeting, event, or community that might be interested in hearing more about US-RSE? We are happy to come talk to your group! Send an email to [contact@us-rse.org](mailto:contact@us-rse.org) or reach out via @sc on slack.

<a name="financials"></a>
# **US-RSE Financials**

> We are organizing US-RSE financials via the [Open Collective](https://opencollective.com/usrse) project. You can see the budget, expenses, and contributions on [this site](https://opencollective.com/usrse).

Reviewer: ```suggestion
We are organizing US-RSE financials via [Open Collective](https://opencollective.com/usrse). You can see the budget, expenses, and contributions on [this site](https://opencollective.com/usrse).
```

> We are organizing US-RSE financials via the [Open Collective](https://opencollective.com/usrse) project. You can see the budget, expenses, and contributions on [this site](https://opencollective.com/usrse).

Reviewer: ```suggestion
We are organizing US-RSE finances via [Open Collective](https://opencollective.com/usrse). You can see the budget, expenses, and contributions on [this site](https://opencollective.com/usrse).
```

You can find [details on the election process of the US-RSE Steering Committee and timeline here](https://us-rse.org/about/election/)! Please don’t hesitate to ask any questions you might have on the [#election](https://usrse.slack.com/archives/C01BC66Q16E) channel on Slack. The voting process will be open from December 3-10, 2021, the election results will be announced by December 17, 2021, and newly elected Steering Committee members will take office January 1, 2022.

<a name="survey"></a>
# **International RSE Survey**

> As part of an international effort, US-RSE has worked with RSE organizations in other countries to develop the [2021 RSE Survey](https://softwaresaved.limequery.com/386272?lang=en). You don’t need to be called an RSE to complete the survey. Your formal job title does not matter: we want to hear from you if you develop software for academic research as part of your work. The survey provides considerable detail about the RSE role: a person’s route into the role, the type of work they conduct, their working practices, job satisfaction and demographics. This means it takes around 20 minutes to complete. We appreciate that this is a significant amount of your time to donate - and we thank you for it!

Reviewer 2: > academic research

Industry R&D is excluded?

Author: Good point. This actually came up. As I understand it the survey has evolved from being targeted at UK academic institutions and has retained at least some of the language to be consistent. That said, I think we should remove the word academic here. It doesn't help.

Author: Removed.

<a name="dei-mc"></a>
# **DEI Media Club Discussion**

In an effort to promote continuing education and discussion, the US-RSE [Diversity, Equity, and Inclusion Working Group (DEI-WG)](https://us-rse.org/about/working-groups/#diversity-equity-and-inclusion-dei) presents the next instance of its DEI Media Club on December 2, 2021, 11:00 AM - 12:00 PM: a discussion of an episode of [Code Switch](https://www.npr.org/podcasts/510312/codeswitch): [The Folk Devil Made Me Do It](https://www.npr.org/2021/08/20/1029775224/the-folk-devil-made-me-do-it).

[Sign up](https://emory.zoom.us/meeting/register/tJAqceCpqTopGtRj9YvJnWD9u3v7SZSdNCBb) to let us know that you are interested.

To get most out of it: Come prepared to discuss!
Please share the event with your colleagues: [https://emory.zoom.us/meeting/register/tJAqceCpqTopGtRj9YvJnWD9u3v7SZSdNCBb](https://emory.zoom.us/meeting/register/tJAqceCpqTopGtRj9YvJnWD9u3v7SZSdNCBb)

<a name="events"></a>
# **Interesting Events**

Some other events coming up may be of interest to RSEs:

> * The [AGU Fall meeting](https://www.agu.org/Fall-Meeting) takes place as a hybrid event December 13-17, 2021 (in person in New Orleans) and has sessions on software and open science such as [Advancing Notebooks as Scholarly Objects](https://agu.confex.com/agu/fm21/meetingapp.cgi/Session/124253).

Reviewer 2: ```suggestion
* The [American Geophysical Union Fall meeting](https://www.agu.org/Fall-Meeting) takes place as a hybrid event December 13-17, 2021 (in person in New Orleans) and has sessions on software and open science such as [Advancing Notebooks as Scholarly Objects](https://agu.confex.com/agu/fm21/meetingapp.cgi/Session/124253).
```

> * [HICSS-55](https://hicss.hawaii.edu/) (55th Hawaii International Conference on System Sciences) is fully virtual, January 3-7, 2022. [Registration](https://hicss-submissions.org/) is required but free! The [Software Technology](https://hicss.hawaii.edu/tracks-55/software-technology/) track includes minitracks such as [Software Sustainability: Strategies for Long-Lasting and Usable Software](https://hicss.hawaii.edu/tracks-55/software-technology/#software-sustainability-strategies-for-long-lasting-and-usable-research-software-minitrack) and [Smart (City) and Data Streaming Application Development: Challenges and Experiences](https://hicss.hawaii.edu/tracks-55/software-technology/#smart-city-and-data-streaming-application-development-challenges-and-experiences-minitrack)**.**

Reviewer 2: ```suggestion
* [HICSS-55](https://hicss.hawaii.edu/) (55th Hawaii International Conference on System Sciences) is fully virtual, January 3-7, 2022. [Registration](https://hicss-submissions.org/) is required but free! The [Software Technology](https://hicss.hawaii.edu/tracks-55/software-technology/) track includes minitracks such as [Software Sustainability: Strategies for Long-Lasting and Usable Software](https://hicss.hawaii.edu/tracks-55/software-technology/#software-sustainability-strategies-for-long-lasting-and-usable-research-software-minitrack) and [Smart (City) and Data Streaming Application Development: Challenges and Experiences](https://hicss.hawaii.edu/tracks-55/software-technology/#smart-city-and-data-streaming-application-development-challenges-and-experiences-minitrack).
```

Author: A goofy script artifact. Good catch!

[Sign up](https://emory.zoom.us/meeting/register/tJAqceCpqTopGtRj9YvJnWD9u3v7SZSdNCBb) to let us know that you are interested.

To get most out of it: Come prepared to discuss!
Please share the event with your colleagues: [https://emory.zoom.us/meeting/register/tJAqceCpqTopGtRj9YvJnWD9u3v7SZSdNCBb](https://emory.zoom.us/meeting/register/tJAqceCpqTopGtRj9YvJnWD9u3v7SZSdNCBb)

<a name="events"></a>
# **Interesting Events**

Some other events coming up may be of interest to RSEs:

* The [AGU Fall meeting](https://www.agu.org/Fall-Meeting) takes place as a hybrid event December 13-17, 2021 (in person in New Orleans) and has sessions on software and open science such as [Advancing Notebooks as Scholarly Objects](https://agu.confex.com/agu/fm21/meetingapp.cgi/Session/124253).

> * [HICSS-55](https://hicss.hawaii.edu/) (55th Hawaii International Conference on System Sciences) is fully virtual ,January 3-7, 2022. [Registration](https://hicss-submissions.org/) is required but free! The [Software Technology](https://hicss.hawaii.edu/tracks-55/software-technology/) track includes minitracks such as [Software Sustainability: Strategies for Long-Lasting and Usable Software](https://hicss.hawaii.edu/tracks-55/software-technology/#software-sustainability-strategies-for-long-lasting-and-usable-research-software-minitrack) and [Smart (City) and Data Streaming Application Development: Challenges and Experiences](https://hicss.hawaii.edu/tracks-55/software-technology/#smart-city-and-data-streaming-application-development-challenges-and-experiences-minitrack)**.**

Reviewer: ```suggestion
* [HICSS-55](https://hicss.hawaii.edu/) (55th Hawaii International Conference on System Sciences) is fully virtual, January 3-7, 2022. [Registration](https://hicss-submissions.org/) is required but free! The [Software Technology](https://hicss.hawaii.edu/tracks-55/software-technology/) track includes minitracks such as [Software Sustainability: Strategies for Long-Lasting and Usable Software](https://hicss.hawaii.edu/tracks-55/software-technology/#software-sustainability-strategies-for-long-lasting-and-usable-research-software-minitrack) and [Smart (City) and Data Streaming Application Development: Challenges and Experiences](https://hicss.hawaii.edu/tracks-55/software-technology/#smart-city-and-data-streaming-application-development-challenges-and-experiences-minitrack)**.**
```

Reviewer: sorry, this was a typo in my suggestion
